en/includes/guides/authentication/conditional-auth/configure-conditional-auth.md

```diff
@@ -29,6 +29,10 @@ There are two ways to add a conditional authentication script:
 ## Add a secret to the script
 Secrets securely store values associated with external APIs. These secret values are used in conditional authentication scripts when {{ product_name }} is required to interact with an external API (service endpoint) during the authentication process. You can securely store these secret values on the {{ product_name }} Console and retrieve them whenever required for `callChoreo()` conditional authentication function.
 
+!!! warning "Before you proceed"
+
+    When working with Asgardeo conditional authentication scripts, <b>never log secrets</b> or sensitive information within your authentication flows.
```

Comment:
Does this section applies only to Asgardeo ? IMO we can have the same statement applicable for Identity Server

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4749/commits/eee0e259cfb15d50c6108cc2add1e85b00b8856d